### PR TITLE
cypress refactor e2e test

### PR DESCRIPTION
**Pull request by wendyvasq**
# Code Task Complete ⚡️

✨ View the Asana Task [here](https://app.asana.com/0/1200099998847559/1203590078157197/f) ✨

## Description of Changes

(A brief idea of what reviewers should be looking for, or the expected behavior of the new update.)

This PR address issue with e2e testing for docs web property. 
The homepage test have been deleted and should be split into individual tests one that focus on the footer and another on the header.

The docs test has been renamed to sideMenuPageLinks.  The primary goal of this test is to ensure that all links in menu are working properly. currently in this PR part of the menu is covered with examples of how to navigate level 1 menu and of the submenu trees.This is an expensive test in execution. I don't recommend this to be part of CI beacuse it takes to long . It should only run when there is changes to the links or the menu. 

From this work there is some discovery that should be to cherry picky what we want to execute for e2e testing 

## Preview Links

https://deploy-preview-2960--getdbt.netlify.app/

## Additional Info

(Any additional comments here, if any)

### CODE REVIEW

File: website/cypress/e2e/sideMenuPageLinks.cy.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+      // failing the test
+      return false;
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/docs/introduction');
+  });
+  it('verifies sidebar menu has the correct level one items', () => {
+    // > li yields liste items
```
Comment (JKarlavige) [inline website/cypress/e2e/sideMenuPageLinks.cy.js]: Just a small typo here `liste`

**Review by JKarlavige — DISMISSED** [review]
Just had one small note on a typo, but all functionality looks to work. Ran the e2e test locally and all tests passed.

Awesome work with this, you really upped the game with our testing!! :shipit:

File: website/cypress/e2e/sideMenuPageLinks.cy.js (hunk 1 above)
Comment (wendyvasq) [reply website/cypress/e2e/sideMenuPageLinks.cy.js]: awesome thank you !

**Review by JKarlavige — APPROVED** [review]
:shipit:

**Pull request merged**